src/main/resources/default/templates/biz/tenants/tenant-config.html.pasta

```diff
@@ -3,7 +3,7 @@
 
 <i:invoke template="/templates/biz/tenants/tenant.html.pasta" tenant="tenant" controller="controller" page="config">
     <t:codeEditor id="editor" mode="json">@tenant.getTenantData().getConfigString()</t:codeEditor>
-    <t:formBar backButton="false"/>
+    <t:formBar backButton="false" singleClick="false"/>
```

Comment:
I think we should rather limit the single-click handling to pages where a form actually exists...:
https://github.com/scireum/sirius-web/blob/4836f26736699091181787d5e4591c25cf9e36b1/src/main/resources/default/assets/tycho/scripts/form.js.pasta#L2C36-L19C8

Reply:
We also should perform a null-check here:
https://github.com/scireum/sirius-web/blob/4836f26736699091181787d5e4591c25cf9e36b1/src/main/resources/default/assets/tycho/scripts/form.js.pasta#L17C1-L17C1

Reply:
> We also should perform a null-check here: https://github.com/scireum/sirius-web/blob/4836f26736699091181787d5e4591c25cf9e36b1/src/main/resources/default/assets/tycho/scripts/form.js.pasta#L17C1-L17C1

We already have a null check here: https://github.com/scireum/sirius-web/blob/4836f26736699091181787d5e4591c25cf9e36b1/src/main/resources/default/assets/common/core.js.pasta#L375



> I think we should rather limit the single-click handling to pages where a form actually exists...: https://github.com/scireum/sirius-web/blob/4836f26736699091181787d5e4591c25cf9e36b1/src/main/resources/default/assets/tycho/scripts/form.js.pasta#L2C36-L19C8

I feel like there could be instances where data i submitted "indirectly" (outside any form like it is the case here) where we want to only be able to submit the data once. I think it's cleaner when singleClick == false always means that the submit action is only triggered once.

Reply:
If there is no form, there is no page reload and having single click == true will always screw the behaviour. On the other hand, if there is a page-reload, why not use a simple form anyway?

TL;DR - no form -> no single-click enforcing seems always right...